gradle-module-plugin/README.md

```diff
@@ -28,16 +28,23 @@ The easiest way to get started with this plugin is to create a new module projec
 
 
-Choosing which [Configuration](https://docs.gradle.org/current/userguide/declaring_dependencies.html) to apply may have important but subtle impacts on your module, as well as your development environment.  Also keep in mind the effect of _project_ dependency configurations.  
+Choosing which [Configuration](https://docs.gradle.org/current/userguide/declaring_dependencies.html) to apply may have important but subtle impacts on your module, as well as your development/build environment.  In general, the following rule of thumb is a good starting point:
 
-> Maven Users: If you're familiar with Maven's dependency scopes, you might initially find Gradle's handling of dependencies to be unnecessarily convoluted.  This is a product of Gradle's powerful (but more complex) dependency management.  We suggest reading the gradle docs on [Working With Dependencies](https://docs.gradle.org/current/userguide/core_dependency_management.html), followed by reading the [Java Library Plugin](https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_separation) documentation.  
+| Configuration  | Usage Suggestion | Included in Module? | Includes Transitive Dependencies? |
+|-------|--------|-------------------------|----------|
+| compileOnly   | Use for 'compile time only' dependencies, including ignition sdk dependencies. Similar to maven 'provided'.  | No | No |
+| compileOnlyApi   | Use for 'compile time only' dependencies, including ignition sdk dependencies. Similar to maven 'provided'.  | No | Yes |
+| api    | Project dependencies that do not explicitly get registered in the module DSL project scopes&#10013;  | No | Yes |
+| implementation   | Project dependencies that do not explicitly get registered in the module DSL project scopes&#10013; | No | No |
+| modlImplementation   | Dependencies that are used in a module project's implementation, but are not part of a public API  | Yes | No|
```

Comment:
the last `No` here ties into my previous comment that it _does_ include them for use at runtime, just not no the compile classpath.